.github/workflows/main.yml

```diff
@@ -5,17 +5,17 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v1
-      - run: docker build -t appliance-build:latest docker
-      - run: ./scripts/docker-run.sh gradle ansibleCheck
+      - run: sudo ./.github/scripts/install-ansible-lint.sh
+      - run: gradle ansibleCheck
```

Comment:
Are we using a preinstalled version of Gradle? If so, we might be better of installing Gradle explicitly, in case the version of the preinstalled one changes.

Reply:
Yes, it is.

I opted for this, since I was having trouble properly setting the `PATH` variable to enable access to both `ansible-lint` and `gradle` installed in `/opt`.

But if this is a requirement (I'll assume it is), I'll give it another shot, and see if I can come up with something relatively clean.

Reply:
Is invoking it by absolute path an option?

Reply:
Yea, that seems reasonable. Pushed a commit to do this.